react/components/Checkbox/index.js

```diff
@@ -119,6 +121,8 @@ Checkbox.propTypes = {
   value: PropTypes.string,
   /** Partial state */
   partial: PropTypes.bool,
+  /** Label size */
+  size: PropTypes.oneOf(['small']),
```

Comment:
Wouldn't it be better if this was a boolean prop named `small`?

Reply:
Not sure @bivillar since in the future we could add a "regular" or "large" for x reasons.

Reply:
I agree with what @kevinch said